tests/apps/lwaftr/data/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# Makefile
+# Adrian Perez, 2015-10-01 14:56
+#
+
+in_pcaps := $(filter-out empty.pcap,$(filter-out %-vlan.pcap,$(wildcard *.pcap)))
+out_pcaps := $(patsubst %.pcap,%-vlan.pcap,$(in_pcaps))
+
+%-vlan.pcap: %.pcap
+	python2 add-dot1q.py $< $@
+
+all: $(out_pcaps)
+
+clean:
+	$(RM) $(out_pcaps)
+
+# vim:ft=make
```

Comment:
Also not a fan of adding this type of information to a source file. Other Makefiles don't have it and I believe it's possible to do this setting in your ~/.vimrc, right?